lbrynet/core/BlobManager.py

```diff
@@ -126,35 +127,17 @@ def _make_new_blob(self, blob_hash, upload_allowed, length=None):
         log.debug('Making a new blob for %s', blob_hash)
         blob = self.blob_type(self.blob_dir, blob_hash, upload_allowed, length)
         self.blobs[blob_hash] = blob
-        d = self._completed_blobs([blob_hash])
-
-        def check_completed(completed_blobs):
-
-            def set_length(length):
-                blob.length = length
-
-            if len(completed_blobs) == 1 and completed_blobs[0] == blob_hash:
-                blob.verified = True
-                inner_d = self._get_blob_length(blob_hash)
-                inner_d.addCallback(set_length)
```

Comment:
The length now gets calculated by the blob itself if its not passed into the HashBlob constructor.  Its only real use is when downloading a blob, you need to keep track of the data received.

I am going to take out `get_blob_length`